scripts/generate-tokens-wrapper.js

```diff
@@ -135,124 +168,195 @@ const targetHost = (css) => {
     return css.replaceAll(spectrumThemeSelectorRegExp, ':host,\n:root {');
 };
 
-const removeImportantComments = (css) => {
-    /**
-     * Spectrum CSS uses /*! comments that are "not" removable.
-     * These comments pile up in merged files, so we _need_ to remove them.
-     */
-    return css.replaceAll(importantCommentRegExp, '');
-};
-
 /**
- * copies @spectrum-css/dist/css/*.css and @spectrum-css/dist/css/**\/*.css
- * replaces classes with :root, :host, and pastes them into
- * corresponding /tools/styles/*.css and /tools/styles/tokens/**\/*.css
- * @param {string} srcPath @spectrum-css/dist/css path
- * @param {*} tokensDir styles/tokens path
+ * Core entry function for generating tokens from Spectrum CSS packages
+ * to be used by the sp-theme
+ * @param {'spectrum'|'spectrum-two'} spectrumVersion
+ * @param {Object} options
+ * @param {boolean} options.clean - Whether to clean the output directory before writing
+ * @returns {Promise}
  */
-const processTokens = (srcPath, tokensDir) => {
-    let css = fs.readFileSync(srcPath, 'utf8');
-    const fileName = srcPath.split(path.sep + 'css' + path.sep).at(-1);
+export async function generateTokensWrapper(
```

Comment:
nit: I don’t have a clear idea of how long these operations usually take. Could we use console.time() to measure it? This might help us start understanding what takes how much time in our build chain.

Reply:
I like the idea a lot - we do this in CSS for our script output as well. It's a bit out of scope here I feel because it's important when setting up the console times that you kick off the timer at the start and close the timer right when it ends but the current structure of this file would make that challenging (having to pass the console key around for instance to ensure it is reported accurately).